packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.css

```diff
@@ -29,6 +30,16 @@
   vertical-align: baseline;
 }
 
+.generated-answer__rephrase--buttons,
+.generated-answer__toggle--button,
+.generated-answer__actions {
+  justify-content: flex-end;
+}
+
+.flex-one{
+  flex: 1;
+}
+
```

Comment:
Why aren't we using the slds classes for things like this?

https://www.lightningdesignsystem.com/utilities/grid/#Content-right-justified ?

Reply:
Indeed this one we could use.